portal_spa/src/pages/RelyingPartiesManagePage.tsx

```diff
@@ -83,7 +83,7 @@ export default function RelyingPartyManager() {
               defaultValues={initialData}
               onSubmit={handleSave}
               serverErrors={fieldErrors}
-              globalError={globalError}
+              globalError={globalError || ""}
```

Comment:
It's perfectly fine to have an undefined value here, right? I would recommend adding `undefined` as a possible value for `globalError` in the `RelyingPartyForm`